<launch>
  <param name="cloud_in_1"  value="/kinect_first/hd/points" />
  <param name="cloud_in_2"  value="/kinect_second/hd/points" />
  <node pkg="motoman_point_cloud" type="point_cloud_merger" name="point_cloud_merger" output="screen">
    <param name="cloud_out"  value="/merged_cloud" />
  </node>
</launch>
